File: javascript/pangram.js
// is there set in javascript?
// if so, i'd use set, throw the sentence into set
// make sentence lowercase, find only letters a-z using regex, case insensitive
// if the result equals 26, return true, if not, false

export const isPangram = (sentence) => {
  if (new Set(sentence.toLowerCase().match(/[a-z]/gi)).size === 26) {
    return true;
  } else {
    return false;
  }
};

File: python/pangram.py
import re 

def is_pangram(sentence):
    # make sentence lowercase
    # use regex and remove all non-lowercase a-z characters
    # if size of set is 26, return true, else false
    sentence = re.sub('[^a-z]','',sentence.lower())
    return True if (len(set(sentence)) == 26) else False
